### ISSUE TITLE
Feature Request: show latency per I/O in verbose mode

### ISSUE THREAD

**Issue by rob-scheepens** [microsoft/diskspd#98]
It would be nice to have verbose mode also list the latency for an I/O.

**Comment by dl2n** [comment]
-L produces the latency distribution. What are you looking for?

If you want per-IO tracing, taking a wpr trace of the run would provide that.

**Comment by rob-scheepens** [comment]
Right, a WPR trace would provide that info, but only after the run. I just would find it handy to have IO stats listed while running. It’s not a big issue, just a nice to have.

**Comment by dl2n** [comment]
I can't think of many situations where actual per-IO stats on stdout wouldn't be overwhelming.

Running per-second/interval aggregates like interval IOPS, average latency, etc. would have some utility and been considered before. There's nothing in the way of that except someone able to contribute, with sufficient time.
